src/lib/util/unicode.cpp

```diff
@@ -8,9 +8,13 @@
 
 ***************************************************************************/
 
+#include <codecvt>
+#include <locale>
+
```

Comment:
Please `#include` standard library headers after project headers, to protect against inadvertently getting into a situation where a header doesn't work if you don't `#include` something else first.